tests/short-first-segment.sh

```diff
@@ -24,8 +24,10 @@ cd "${SCRATCH}"
 
 ldd "${EXEC_NAME}"
 
-${PATCHELF} --add-rpath lalalalalalalala --output modified1 "${EXEC_NAME}"
+${PATCHELF} --set-rpath "$(tr -dc A-Za-z0-9 </dev/urandom | head -c 4096)" --output modified1 "${EXEC_NAME}"
```

Comment:
I would make the test reproducible and not have a random value here i.e. storing a static secret somewhere.